CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ macro(build_benchmark)
 
   externalproject_add(benchmark-${GOOGLE_BENCHMARK_TARGET_VERSION}
     URL "https://github.com/google/benchmark/archive/v${GOOGLE_BENCHMARK_TARGET_VERSION}/benchmark-${GOOGLE_BENCHMARK_TARGET_VERSION}.tar.gz"
-    URL_MD5 91d2d9a824cab82c67a80ccce5b93218
+    URL_MD5 084b34aceaeac11a6607d35220ca2efa
```

Comment:
The hash doesn't seem to match the one from the 1.5.2 release (https://github.com/google/benchmark/releases)?

@cottsay Weiddly the previous hash doesn't match the one from the 1.5.1 release either?

Reply:
its a file md5sum vs a commit tag

Reply:
Ups...